apartment/api/views.py

```diff
@@ -32,12 +33,13 @@ class ProjectAPIView(APIView):
 
     def get(self, request, project_uuid=None):
         many = project_uuid is None
-        project_data = get_projects(project_uuid)
-        if not many:
-            if len(project_data) == 1:
+        project_data = None
+        try:
+            project_data = get_projects(project_uuid)
+            if not many:
                 project_data = project_data[0]
-            else:
-                raise NotFound()
+        except ObjectDoesNotExist:
```

Comment:
This is perhaps a bit opinionated thing, but I would rather use some own custom exception for this, because `ObjectDoesNotExist` normally related to Django model objects, which aren't playing any part here.

Reply:
I have a different opinion here. `ObjectDoesNotExist` is a Django exception that can utilize on objects and it is not related to Django models anyhow. Django has an inheritance from the `ObjectDoesNotExist` which has been separately specified for the models.

Reply:
Makes sense 👍